docker/kubernetes-tentacle/bootstrapRunner/bootstrapRunner.go

```diff
@@ -79,6 +79,9 @@ func main() {
 		fmt.Fprintln(os.Stderr, "bootstrapRunner.go: Failed to execute bootstrap script", err)
 	}
 
+	stdoutLogFile.Flush()
+	stderrLogFile.Flush()
```

Comment:
Do we need to do anything about the error returned?

Reply:
hrm, I thought about that, but I figured the program was existing anyway so you kinda don't care 🤔

Reply:
I suppose we could try printing to stderr so there is a chance it can be seen somewhere. The most likely error that I can think of would be "no space left on device"

Reply:
I agree - it's worth capturing and logging the error explicitly, even if it's just to let people looking at the code know that it _can_ fail!

Reply:
fair, I've updated them to write to stderr if it fails to flush